src/routes/settings/Ollama.svelte

```diff
@@ -125,42 +126,49 @@
 			<Badge
 				variant={$settingsStore.ollamaServerStatus === 'disconnected' ? 'warning' : 'positive'}
 			>
-				{$i18n.t(`${$settingsStore.ollamaServerStatus}`)}
+				{#if $settingsStore.ollamaServerStatus === 'disconnected'}
+					{$LL.disconnected()}
+				{:else}
+					{$LL.connected()}
+				{/if}
 			</Badge>
 		</svelte:fragment>
 
 		<svelte:fragment slot="help">
 			{#if ollamaURL && $settingsStore.ollamaServerStatus === 'disconnected'}
 				<FieldHelp>
 					<P>
-						{$i18n.t('allowConnections')}
+						{$LL.allowConnections()}
 						<Badge capitalize={false}>{ollamaURL.origin}</Badge>
 						<Button
 							variant="link"
 							href="https://github.com/jmorganca/ollama/blob/main/docs/faq.md#how-can-i-allow-additional-web-origins-to-access-ollama"
 							target="_blank"
 						>
-							{$i18n.t('seeDocs')}
+							{$LL.seeDocs()}
 						</Button>
 					</P>
 					<P>
-						{$i18n.t('checkBrowserExtensions')}
+						{$LL.checkBrowserExtensions()}
 					</P>
 					{#if ollamaURL.protocol === 'https:'}
 						<P>
 							<!-- eslint-disable-next-line svelte/no-at-html-tags -->
-							{@html $i18n.t('tryingToConnectNotLocalhost')}
+							{@html $LL.tryingToConnectNotLocalhost({
+								hostname: '<code class="badge">localhost</code>',
+								ip: '<code class="badge">127.0.0.1</code>'
+							})}
```

Comment:
Apparently there is a way to render components in the middle of a localized string but [we have to implement it ourselves](https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/adapter-svelte#how-do-i-render-a-component-inside-a-translation). I don't think we should bother with that until we have many such cases.

In the meantime I figured it's best keeping all of our markup in the Svelte component and simply interpolate the HTML as a string. I don't think this approach will work in a lot of cases, but we'll cross that bridge when we have to.